llama.cpp

```diff
@@ -3122,9 +3122,9 @@ size_t llama_copy_state_data(struct llama_context * ctx, uint8_t * dst) {
         if (kv_size) {
             const size_t elt_size = ggml_element_size(kv_self.k);
 
-            char buffer[4096];
+            //char buffer[4096];
```

Comment:
I don't see much reason to leave this here, let's remove this comment.

Reply:
Done.